integration_tests/integration_dotnet_test.go

```diff
@@ -301,6 +301,19 @@ func TestLargeResourceDotNet(t *testing.T) {
 	})
 }
 
+// tests that when a resource transformation throws an exception, the program exists
```

Comment:
I make this typo all the time :D

```suggestion
// tests that when a resource transformation throws an exception, the program exits
```